src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/custom.py

```diff
@@ -159,95 +174,64 @@ def create_application_package(client,
 
 # Data plane custom commands
 
-def _handle_batch_exception(action):
-    try:
-        return action()
-    except BatchErrorException as ex:
-        try:
-            message = ex.error.message.value
-            if ex.error.values:
-                for detail in ex.error.values:
-                    message += "\n{}: {}".format(detail.key, detail.value)
-            raise CLIError(message)
-        except AttributeError:
-            raise CLIError(ex)
-    except (ValidationError, ClientRequestError) as ex:
-        raise CLIError(ex)
-
 
 @transfer_doc(CertificateAddParameter)
-def create_certificate(client, certificate_file, thumbprint, password=None):
+def create_certificate(cmd, client, certificate_file, thumbprint, password=None):
     thumbprint_algorithm = 'sha1'
-
-    def action():
-        client.add(cert)
-        return client.get(thumbprint_algorithm, thumbprint)
-
     certificate_format = 'pfx' if password else 'cer'
     with open(certificate_file, "rb") as f:
         data_bytes = f.read()
     data = base64.b64encode(data_bytes).decode('utf-8')
     cert = CertificateAddParameter(thumbprint, thumbprint_algorithm, data,
                                    certificate_format=certificate_format,
                                    password=password)
-    return _handle_batch_exception(action)
+    client.add(cert)
+    return client.get(thumbprint_algorithm, thumbprint)
 
 
-def delete_certificate(client, thumbprint, abort=False):
+def delete_certificate(cmd, client, thumbprint, abort=False):
```

Comment:
For a lot of your custom commands, it doesn't look like you use/need `cmd`.  You can remove it, or you'd need to add a pylint exception so it won't fail due to the unused variable.